feat(js): Add note about cookies to troubleshooting guide (#5779)
getsentry/sentry-docs: src/platforms/javascript/common/troubleshooting/index.mdx

 Access-Control-Allow-Origin: *
```

## Sentry Cookies

> If you are testing your setup and see Sentry cookies in your local cookie storage, note that these cookies are NOT set by any Sentry SDK, and therefore will not be set on your users' machines. Those cookies come from the Sentry frontend, and do things like allow you to stay logged in for a period of time after you close a tab containing Sentry.
> 
> If you instead use an incognito window to send test events to Sentry, it will mimic more closely what will happen for your users (assuming they are not themselves Sentry customers, of course!), and you can confirm that no Sentry cookies are set.

Reviewer: ```suggestion
If you're testing your setup and see Sentry cookies in your local cookie storage, note that they're NOT set by any Sentry SDK and won't be set on your users' machines. Sentry cookies come from our frontend, and do things like allow you to stay logged in for a period of time after you close a Sentry tab.

If you want to see what your users see, (assuming they're not Sentry customers), use an incognito window to send test events to Sentry instead. That way, you can confirm that no Sentry cookies are set.
```

Reviewer 2: Do the SDKs send the Sentry cookies if the user visiting the website is a Sentry customer (and logged in)? I would not expect this to be the case.

Since the cookies are scoped to our domain (`.sentry.io`), I would expect the user's request to the website utilizing the Sentry SDK to only include relevant cookies for said website.
